Tests/swift-sdk-swift-tests/InboxImpressionTrackerTests.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  InboxImpressionTrackerTests.swift
+//  swift-sdk-swift-tests
+//
+//  Created by Jay Kim on 12/3/19.
+//  Copyright © 2019 Iterable. All rights reserved.
+//
+
+import XCTest
+
+@testable import IterableSDK
+
+class InboxImpressionTrackerTests: XCTestCase {
+    func testToIterableInboxImpression() {
+        let messageId = IterableUtil.generateUUID()
+        let silentInbox = true
+        let displayCount = 5
+        let duration = 7.31
+        
+        let inboxImpression = IterableInboxImpression(messageId: messageId,
+                                                      silentInbox: silentInbox,
+                                                      displayCount: displayCount,
+                                                      displayDuration: duration)
+        
+        let convertedImpression = InboxImpressionTracker.Impression(messageId: messageId,
+                                                                    silentInbox: silentInbox,
+                                                                    displayCount: displayCount,
+                                                                    duration: duration).toIterableInboxImpression()
+        
+        XCTAssertEqual(inboxImpression.messageId, convertedImpression.messageId)
+        XCTAssertEqual(inboxImpression.silentInbox, convertedImpression.silentInbox)
+        XCTAssertEqual(inboxImpression.displayCount, convertedImpression.displayCount)
+        XCTAssertEqual(inboxImpression.displayDuration, convertedImpression.displayDuration)
+    }
+    
+    func testUpdateVisibleRows() {
+        let rowInfo1 = InboxImpressionTracker.RowInfo(messageId: IterableUtil.generateUUID(), silentInbox: false)
+        let rowInfo2 = InboxImpressionTracker.RowInfo(messageId: IterableUtil.generateUUID(), silentInbox: true)
+        let rowInfo3 = InboxImpressionTracker.RowInfo(messageId: IterableUtil.generateUUID(), silentInbox: false)
+        
+        let tracker = InboxImpressionTracker()
+        
+        tracker.updateVisibleRows(visibleRows: [rowInfo1])
+        
+        sleep(30)
```

Comment:
Is the time in seconds or milliseconds? If seconds this is too long for a test.

Reply:
Misread what the impression minimum is while I wrote this, haha. Will fix. `sleep` uses seconds.